src/traits/style_provider.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013-2015, The Rust-GNOME Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::FFIGObject;
+
+/// The priority used for default style information that is used in the absence of themes.
+/// Note that this is not very useful for providing default styling for custom style classes,
+/// themes are likely to override styling provided at this priority with catch-all * {...} rules.
+pub const PRIORITY_FALLBACK: u32 = 1;
```

Comment:
The idiomatic approach would be to define `GTK_STYLE_PROVIDER_PRIORITY_*` constants in the `sys` crate (those definitions will become autogenerated eventually) and use those on the right hand side here.